components/permissions/contexts/brave_social_media_permission_context.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PERMISSIONS_CONTEXTS_BRAVE_SOCIAL_MEDIA_PERMISSION_CONTEXT_H_
+#define BRAVE_COMPONENTS_PERMISSIONS_CONTEXTS_BRAVE_SOCIAL_MEDIA_PERMISSION_CONTEXT_H_
+
+#include "components/permissions/permission_context_base.h"
+
+namespace permissions {
+
+class BraveSocialMediaPermissionContext : public PermissionContextBase {
```

Comment:
why `SocialMedia`? `Media` in Chromium usually means camera/mic/video/audio, etc.
this class is used only for `ContentSettingsType::BRAVE_GOOGLE_SIGN_IN` type.

Reply:
Sorry this was leftover from old code, changing name to `BraveGoogleSignInPermissionContext`